pkg/yaml2pcl/yaml2pcl.go

```diff
@@ -25,11 +26,14 @@ import (
 // }
 // }
 func ConvertFile(filename string) (string, error) {
-	testFiles, err := parser.ParseFile(filename, parser.ParseComments)
-	if err != nil {
-		return "", err
+	if strings.Contains(filename, ".yaml") || strings.Contains(filename, ".yml") {
+		testFiles, err := parser.ParseFile(filename, parser.ParseComments)
+		if err != nil {
+			return "", err
+		}
+		return convert(*testFiles)
 	}
-	return convert(*testFiles)
+	return "", fmt.Errorf("input was not a YAML file: %s", filename)
```

Comment:
This message is a little misleading. The logic requires that the file has a yaml extension, but doesn't verify the contents.

We should either update the message to indicate that the file didn't have the expected extension, or change the logic to try each file and bail out if it doesn't parse correctly.

Reply:
I agree. We should be concerned about whether or not the file contents parse and not be opinionated about the extension